packages/app/src/pages/dashboard/endpoints/index.tsx

```diff
@@ -29,7 +31,7 @@ const DashboardEndpointsPage: React.FC<Props> = () => {
     <>
       <Metadata title="Dashboard" />
       <Layout
-        renderAppBar={renderAppBar}
+        renderAppBar={lg ? undefined : renderAppBar}
```

Comment:
When the value is not undefined, Layout renders the appbar and make space for it,  so if you don't want to render the appBar, you need to pass undefined.

Neither null nor false is acceptable. 
It may need to be refactored, but not now.